taqueria-protocol/Config.ts

```diff
@@ -0,0 +1,92 @@
+import {z} from 'zod'
+import * as InstalledPlugin from '@taqueria/protocol/InstalledPlugin'
+import * as NetworkConfig from '@taqueria/protocol/NetworkConfig'
+import * as SandboxConfig from "@taqueria/protocol/SandboxConfig"
+import * as Environment from "@taqueria/protocol/Environment"
+import * as Tz from "@taqueria/protocol/Tz"
+
+const networkMap = z.record(
```

Comment:
Not sure if it was intentional, but it looks like the default config is missing the `network` property after the refactor:
```
{
    "language": "en",
    "testsDir": "tests",
    "contractsDir": "contracts",
    "artifactsDir": "artifacts",
    "sandbox": {
        "local": {
            "label": "Local Tezos Sandbox",
            "rpcUrl": "http://localhost:20000",
            "protocol": "PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx"
        }
    },
    "environment": {
        "default": "development",
        "development": {
            "networks": [],
            "sandboxes": [
                "local"
            ]
        }
    },
    "accounts": {
        "bob": "3_000_000_000",
        "alice": "3_000_000_000",
        "john": "3_000_000_000",
        "jane": "3_000_000_000",
        "joe": "3_000_000_000"
    }
}
```